marklogic-data-hub/src/server-side/impl/trace-lib.xqy

```diff
@@ -642,12 +641,11 @@ declare function trace:get-traceIds($q as xs:string?)
       cts:element-value-query(xs:QName("identifier"), fn:lower-case($q) || "*", "wildcarded")
     else ()
   let $results :=
-    cts:element-value-co-occurrences(
-      xs:QName("traceId"),
-      xs:QName("identifier"),
+    cts:value-co-occurrences(
+      cts:path-reference("/trace/traceId"),
+      cts:path-reference("/trace/identifier"),
```

Comment:
has the traceDB config been updated to add this path range index?

Reply:
ditto